vision/google/cloud/vision/image.py

```diff
@@ -30,22 +30,28 @@ class Image(object):
     :type content: bytes
     :param content: Byte stream of an image.
 
+    :type filename: str
+    :param filename: Filename to image.
+
     :type source_uri: str
     :param source_uri: Google Cloud Storage URI of image.
 
     :type client: :class:`~google.cloud.vision.client.Client`
     :param client: Instance of Vision client.
     """
 
-    def __init__(self, client, content=None, source_uri=None):
+    def __init__(self, client, content=None, filename=None, source_uri=None):
```

Comment:
I added this in so we could look at it. Also I added the tests as well to look at.

https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2778/commits/a431df1373f7cc0021186bdffc89bfe1dcc0ec08